spec/plaindatetime.html

```diff
@@ -1157,5 +1128,34 @@ <h1>
         1. Return ! CreateDurationRecord(_dateDifference_.[[Years]], _dateDifference_.[[Months]], _dateDifference_.[[Weeks]], _balanceResult_.[[Days]], _balanceResult_.[[Hours]], _balanceResult_.[[Minutes]], _balanceResult_.[[Seconds]], _balanceResult_.[[Milliseconds]], _balanceResult_.[[Microseconds]], _balanceResult_.[[Nanoseconds]]).
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalplaindatetime" aoid="DifferenceTemporalPlainDateTime">
+      <h1>
+        DifferenceTemporalPlainDateTime (
+          _direction_: 1 or -1,
+          _dateTime_: a Temporal.PlainDateTime,
+          _other_: an ECMAScript language value,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate options as an ECMAScript language value for some type but not the other types?